includes/features/drupal.org/addtocal/addtocal.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Add to Cal administration module
+  In order to ensure the site is functioning as expected
+  As administrator
+  I want to verify functionality of the Add to Cal module
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+
+  @safe @api
+  Scenario: Verify module is enabled
+    And I am on "admin/modules"
+    Then I should see 1 "#edit-modules-other-addtocal-enable" element
+    And the "modules[Other][addtocal][enable]" checkbox should be checked
+
+  @dev @api @javascript @destructive
+  Scenario: Verify module functionality
+    Given the "addtocal" module is enabled
+    And the cache has been cleared
+    Given I am on "admin/structure/types/manage/article/fields"
```

Comment:
Tests cannot rely on the article content type as it is not on all sites. You will need to create a new test content type in the test to ensure that you can attach a field to it.